pkgs/racket-doc/scribblings/more/more.scrbl

```diff
@@ -95,6 +95,8 @@ start @exec{racket} with no command-line arguments:
   > 
 }
 
+@margin-note*{Set your @tt{PATH} environment variable so you can use raco and other Racket command line functions. On macOS:@tt{sudo sh -c 'echo "/Applications/Racket v7.7/bin" >> /etc/paths.d/racket'}. On Windows add the racket @tt{bin} path to @tt{Path} in 'Environment Variables' (under System Properties, Advanced tab)}
```

Comment:
1. I think it's better to use `margin-note` rather than `margin-note*`
2. Use `@exec{raco}`.
3. I think Racket people prefer MacOS over macOS
4. Add a space after `macOS:`
5. Add a `:` after `Windows`.
6. Users might not know that they need to change `7.7` to the current version of Racket. We also shouldn't hard code a version number since we need to change it every release. I'm not sure what placeholder we should put there.

Reply:
You can produce the current version number with the `version` function.

Reply:
I was hoping for something like `echo "/Applications/Racket v<racket version number>/bin"` with a different font for `<racket version number>` to make it clear that it's a placeholder that users need to fill in. But I guess the `version` function is a simpler solution.

Reply:
Perhaps also worth noting: the program isn't guaranteed to be in /Applications, even if people installed one of the standard releases. They are encouraged to drag it there, but they might drag it to the Desktop or they might just run it on the mounted dmg (I know the latter is a real thing as there was a bug that affected only people like that and we got reports....took a while to figure out that the dmg thing was relevant!)

Reply:
Made all suggested changes. Please review and let me know if you see any issues.